.github/ISSUE_TEMPLATE/config.yml

```
@@ -1,5 +1,5 @@
 blank_issues_enabled: false
 contact_links:
   - name: I need help
-    url: https://devdocs.prestashop.com/1.7/faq/i-need-help/
+    url: https://devdocs.prestashop.com/8/faq/i-need-help/
```

I think this link makes more sense
```suggestion
    url: https://www.prestashop-project.org/support/
```